README.md

```
@@ -54,10 +55,10 @@ password handling and robust validation.
 
 ## Data Format
 
-### File Storage Format (users_db.txt)
+### File Storage Format (remote PostgreSQL)
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)